x-pack/plugins/lens/public/editor_frame_plugin/plugin.tsx

```diff
@@ -23,19 +29,30 @@ class EditorFramePlugin {
       render: domElement => {
         this.element = domElement;
```

Comment:
If we don't follow the `createInstance` proposal form above (and I think we should), we have to check whether render was already called on another element and clean up there:
```ts
if (this.element !== null && this.element !== domElement) {
  unmountComponentAtNode(this.element);
}
```